packages/datadog-instrumentations/test/passport-http.spec.js

```diff
@@ -85,6 +103,18 @@ withVersions('passport-http', 'passport-http', version => {
       return agent.close({ ritmReset: false })
     })
 
+    it('should not call subscriber when an error occurs', async () => {
+      const res = await axios.get(`http://localhost:${port}/`, {
+        headers: {
+          // error:1234
+          Authorization: 'Basic ZXJyb3I6MTIzNAo='
```

Comment:
Ah whoops, that's my bad, I did a `echo "error:1234" | base64` and forgot it would include the newline. (it doesn't fail the test because we don't check for the password in an error case